packages/devtools_app/lib/src/timeline/timeline_processor.dart

```diff
@@ -603,7 +603,7 @@ class FullTimelineProcessor extends TimelineProcessor {
       : super(timelineController);
 
   /// Number of traceEvents we will process in each batch.
-  static const _defaultBatchSize = 250;
+  static const _defaultBatchSize = 2000;
```

Comment:
Should this number be based on the delay for batchProcessing or does it not matter?

Reply:
The delay is more important for the appearance of the progress indicator. This number should be whatever makes sense for the type of data being processed. The number of trace events we can be in the tens of thousands, so we want the data to be able to process quickly. I manually tested a release build for several different batch sizes and delays, and this is what I settled on to provide quick processing and a smooth progress indicator. With the previous size of 250, it added ~10 seconds of processing time to a typical trace from dream (g3), which is how I discovered that these numbers needed to be modified.